schema.json

```diff
@@ -0,0 +1,204 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/scverse/ecosystem-packages/main/schema.json",
+    "title": "Scverse Ecosystem Packages",
+    "description": "An approved scverse ecosystem package that fulfils some minimal requirements",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "name": {
+                "description": "The name of the package. Must be unique",
+                "type": "string"
+            },
+            "description": {
+                "description": "Summary of the purpose of the package (1-2 sentences).",
+                "type": "string"
+            },
+            "project_home": {
+                "description": "Link to the source code of the project (Usually a GitHub project page)",
+                "type": "string",
+                "format": "uri"
+            },
+            "documentation_home": {
+                "descripton": "Link to the documentation of the package (e.g. readthedocs)",
+                "type": "string",
+                "format": "uri"
+            },
+            "tutorials_home": {
+                "descripton": "Link to the tutorials/vignette (often a subpage of the documentation)",
+                "type": "string",
+                "format": "uri"
+            },
+            "install": {
+                "type": "object",
+                "items": {
+                    "properties": {
+                        "pypi": {
+                            "description": "PyPI identifier",
+                            "type": "string"
+                        },
+                        "conda": {
+                            "description": "Conda identifier, including channel (e.g. conda-forge::scanpy)",
+                            "type": "string",
+                            "pattern": "(.*)::(.*)"
+                        }
+                    }
```

Comment:
For now only PyPI or conda. We can add more entries here as we start adding our first Julia/CLI/... package